test/test-clang-cl.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Huang Qinjin
```

Comment:
Your name.

Reply:
Ok, fair enough. These files were mostly boilerplate from your creations anyway, but I can change it in the newly created files.